misc/review/MidiServiceTransportPluginInfo.idl

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License
+// ============================================================================
+// This is part of the Windows MIDI Services App API and should be used
+// in your Windows application via an official binary distribution.
+// Further information: https://github.com/microsoft/MIDI/
+// ============================================================================
+
+#include "namespace_defines.h"
+#include "midl_defines.h"
+MIDI_IDL_IMPORT
+
+namespace MIDI_MIDL_NAMESPACE
+{
+    MIDI_API_CONTRACT_v1
+    [default_interface]
+    runtimeclass MidiServiceTransportPluginInfo
+    {
+        Guid Id{ get; };
+
+        String Name{ get; };
+        String Mnemonic{ get; };
+        String Description{ get; };
+
+        String SmallImagePath{ get; };
+
+        String Author{ get; };
+
+        String Version{ get; };
+
+        Boolean IsRuntimeCreatableByApps{ get; };
+        Boolean IsRuntimeCreatableBySettings{ get; };
+
+        Boolean IsSystemManaged{ get; };
+        Boolean IsClientConfigurable{ get; };
```

Comment:
"IsClientConfigurable" could mean
* Can the client be configured?
* Can this transport plugin be configured by the client?
Should this be "CanConfigure"? (Assuming the caller is the client.)

Reply:
It should be CanConfigure. Thanks.